integration-tests/src/test/java/oracle/kubernetes/operator/utils/DbUtils.java

```diff
@@ -48,70 +49,149 @@ public static OracleDB createOracleDB(String dbPropsFile) throws Exception {
+  public static void stopOracleDB(String scriptsDir, String dbNamespace) throws Exception {
     String cmd = "sh " 
         + scriptsDir
-        + "/scripts/create-oracle-db-service/stop-db-service.sh";
-    TestUtils.exec(cmd, true);
+        + "/scripts/create-oracle-db-service/stop-db-service.sh"
+        + " -n " + dbNamespace;
+    try {
+      TestUtils.exec(cmd, true);
+    } catch (Exception ex) {
+      ex.getCause();
+    } 
   }
   
   /**
-   * create Oracle rcu pod and load database schema in the k8s cluster default namespace.
-   * @param rcuSchemaPrefix - rcu SchemaPrefixe
-   * @throws Exception - if any error occurs when creating Oracle rcu pod
+   * Create Oracle rcu pod and load database schema in the k8s cluster default namespace.
+   * 
+   * @param scriptDir script dir
+   * @param rcuSchemaPrefix rcu SchemaPrefixe
+   * @param dbPort NodePort of DB
+   * @param dbnamespace namesspace that DB instance is going to start
+   * @throws Exception if any error occurs when creating Oracle rcu pod
    */
-  public static void createRcuSchema(String scriptsDir, String rcuSchemaPrefix) throws Exception {
-    String cmd = "sh " 
+  public static void createRcuSchema(String scriptsDir, String rcuSchemaPrefix, 
+      String dbUrl, String dbNamespace) throws Exception {
+    String cmd;
+    if (dbUrl == null) {
+      cmd = "sh " 
         + scriptsDir
         + "/scripts/create-rcu-schema/create-rcu-schema.sh -s "
         + rcuSchemaPrefix
-        + " -i "
-        + BaseTest.getfmwImageName() + ":" + BaseTest.getfmwImageTag();
-    TestUtils.exec(cmd, true);
+        + " -i " + BaseTest.getfmwImageName() + ":" + BaseTest.getfmwImageTag()
+        + " -n " + dbNamespace;
+    } else {
+      cmd = "sh " 
+          + scriptsDir
+          + "/scripts/create-rcu-schema/create-rcu-schema.sh -s "
+          + rcuSchemaPrefix
+          + " -d "
+          + dbUrl
+          + " -i " + BaseTest.getfmwImageName() + ":" + BaseTest.getfmwImageTag()
+          + " -n " + dbNamespace;
+    }
+    
+    try {
+      TestUtils.exec(cmd, true);
+    } catch (Exception ex) {
+      ex.getCause();
+    } 
+  }
+  
+  /**
+   * Create Docker Registry Secret for the DB namespace.
+   * 
+   * @param namespace namespace where the docker registry secreted is going to create
+   * @throws Exception when the kubectl create secret command fails
+   */
+  public static void createDockerRegistrySecret(String namespace) throws Exception {
+    String secret = System.getenv("IMAGE_PULL_SECRET_FMWINFRA");
+    if (secret == null) {
+      secret = "docker-store";
+    }
+    String ocrserver = System.getenv("OCR_SERVER");
+    if (ocrserver == null) {
+      ocrserver = "container-registry.oracle.com";
+    }
+    try {
+      TestUtils.createDockerRegistrySecret(
+          secret,
+          ocrserver,
+          System.getenv("OCR_USERNAME"),
+          System.getenv("OCR_PASSWORD"),
+          System.getenv("OCR_USERNAME") + "@oracle.com",
```

Comment:
this seems wrong - the env var almost certainly has the entire username in it, and does not need something added to the end

Reply:
I remember previously we need this extra "oracle.com" on external Jenkins and it didn't hurt for local run either. This time it works with or without "oracle.com" . I removed it.